src/bp/sdk/botpress.d.ts

```diff
@@ -632,24 +632,20 @@ declare module 'botpress/sdk' {
       end: number
     }
 
-    export type SlotCollection = Dic<Slot>
-
-    export interface Predictions {
-      [context: string]: {
+    export interface ContextPrediction {
+      confidence: number
+      oos: number
+      intents: {
+        label: string
         confidence: number
-        oos: number
-        intents: {
-          label: string
-          confidence: number
-          slots: SlotCollection
-          extractor: 'exact-matcher' | 'classifier'
-        }[]
-      }
+        slots: Dic<Slot>
```

Comment:
This really tickles me...

When presenting any collection of information for a given type (let's say `T`) we always have two options:

1. 
```ts
({ name: string } & T)[]    // array of the type with key (here "name")
```
2. 
```ts
{
    [name: string]: T    // dictionnary of the type
}
```

Option 1. allows dupplicated values, while option 2. doesnt.

I really don't care about the one we pick and I have no preference at all, BUT let's pick a side.

Here, we have a dictionnary of slots inside an array of intents inside a dictionnary of context predictions...

This is not even mentioning the fact that, for some reason, intents are identified by the field `label` instead of `name` like everywhere else.

Also, why is there 2 differents `Intent` used at predict time?

This is super painfull to refactor as users probably have a ton of calls like `Object.keys(event.nlu.predictions)` in their actions and hooks, 

but by at least agreeing on a standard, we could gradually move toward a more uniform representation of data.